adobe_vipm/flows/utils.py

```diff
@@ -702,6 +706,34 @@ def notify_missing_prices(agreement_id, missing_skus, product_id, currency, comm
     )
 
 
+def notify_not_updated_subscriptions(order_id, error_message, updated_subscriptions, product_id):
+    """
+    Notifies about SKUs with missing prices in the agreement.
+    Args:
+        agreement_id (str): The agreement ID
+        missing_skus (list): List of SKUs without prices
+        product_id (str): The product ID
+        currency (str): The currency code
+        commitment_date (str, optional): The 3YC commitment date if applicable
+    """
+    message = (
+        f"{error_message}\n\n"
+        f"The order **{order_id}**\n\n"
+        f"has failed changing the auto-renewal status\n\n "
+        f"- Product ID: {product_id}\n\n"
+    )
+
+    if updated_subscriptions:
+        message += "The following subscriptions has been updated and rolled back:\n"
+        for subscritpion in updated_subscriptions:
+            message += f"  - {subscritpion["subscription_vendor_id"]}\n"
+
+    send_exception(
+        "Subscription not updated",
```

Comment:
I would suggest to send something meaningful here as a title of the message. At least to add order configuration context